apps/coscrad-frontend/src/components/live/live.tsx

```diff
@@ -0,0 +1,24 @@
+import { MediaPlayer } from '@coscrad/media-player';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+
+export interface AboutProps {
+    about: string;
+}
+
+export const Live = (): JSX.Element => {
+    const {
+        listenLive: { link, playingMessage, title, logoUrl, missionStatement },
+    } = useContext(ConfigurableContentContext);
+
+    return (
+        <div>
+            {/* TODO update media player to display play message */}
```

Comment:
Link this comment to your story for this work.

Reply:
[Here](https://github.com/COSCRAD/coscrad/pull/308#discussion_r1120788458) is the story.